src/Client/app/(admin)/manage-users/page.tsx

```
@@ -32,104 +35,161 @@ const columns = [
       return (
-        <div className="flex gap-4 ml-16">
-          <button aria-label="Edit User" onClick={() => setIsEditModalOpen(true)}>
-            <FaEdit className={'text-xl'} />
+        <div className="flex w-[100px] gap-5 justify-between ml-6">
+          <button
+            aria-label="Change Status"
+            onClick={() => {
+              setSelectedUser(user.accountId)
+              setIsActiveModalOpen(true)
+            }}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)